### PR TITLE
chore: release v1.0.3

### PR DESCRIPTION
**Pull request by Panaetius**
Release v1.0.3

/deploy renku=1.0-next renku-graph=development renku-ui=master

### CODE REVIEW

File: CHANGES.rst [hunk 1]
```diff
@@ -18,6 +18,64 @@
...
+-  **core:** fix Zenodo dataset import if ``sameAs`` is set
+   (`#2572 <https://github.com/SwissDataScienceCenter/renku-python/issues/2572>`__)
+   (`f704916 <https://github.com/SwissDataScienceCenter/renku-python/commit/f7049165b53c69776a5a0a9d2c5ef0fd7b233b62>`__)
+-  **core:** make activity ids deterministic in migration
+   (`#2581 <https://github.com/SwissDataScienceCenter/renku-python/issues/2581>`__)
+   (`7ed6102 <https://github.com/SwissDataScienceCenter/renku-python/commit/7ed6102496abb03329f6b19521232215e31a834a>`__)
+-  **core:** move NodeJS requirement check to cwltool plugin
+   (`#2586 <https://github.com/SwissDataScienceCenter/renku-python/issues/2586>`__)
+   (`1d79ce2 <https://github.com/SwissDataScienceCenter/renku-python/commit/1d79ce27d7661e59e2ddc33b90e6003b16a4e090>`__)
+-  **core:** reintroduce shell completion command
```
Comment (mohammad-sdsc) [inline CHANGES.rst:47]: This seems to be a feature.

File: CHANGES.rst [hunk 2]
```diff
@@ -18,6 +18,64 @@
...
+   (`#2562 <https://github.com/SwissDataScienceCenter/renku-python/issues/2562>`__)
+   (`6b3ee60 <https://github.com/SwissDataScienceCenter/renku-python/commit/6b3ee604db4e23bd8b51a323ec0af93fda0d23ff>`__)
+-  **service:** disable migration check optimization
```
Comment (mohammad-sdsc) [inline CHANGES.rst:50]: I believe this a bit detailed and technical for a change log.

**Review by mohammad-sdsc — COMMENTED** [review]
Thanks Ralf! Just two minor comments regarding the change log. I was wondering if we should update `__version__` in `renku/version.py` as well.

**Comment by Panaetius** [comment]
> I was wondering if we should update `__version__` in `renku/version.py` as well.

https://github.com/SwissDataScienceCenter/renku-python/blob/develop/pyproject.toml#L277 this takes care of that automatically.

**Review by mohammad-sdsc — APPROVED** [review]
🚀

**Pull request merged**